scripts/vagrant-linux-priv-go.sh

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
 function install_go() {
-	local go_version=1.11.1
+	local go_version=1.13.4
```

Comment:
this also works: 
```suggestion
	local go_version=1.13
```
I think it would be a bit better in case of a security patch or something

Reply:
:+1: